cms/static/js/spec/views/edit_chapter_spec.js

```diff
@@ -0,0 +1,27 @@
+define(['jquery', 'underscore', 'js/views/edit_chapter', 'js/models/course'],
+    function ($, _, EditChapter, Course) {
+    'use strict';
+
+        describe('EditChapter', function () {
+
+                beforeEach(function() {
+                    window.course = new Course({
```

Comment:
You shouldn't need to create anything on the window object. You can define variables within a spec (initialized in beforeEach) that test points can reference.